### PR TITLE
latest of zipkin

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

Fixes #

## Proposed Changes

- updating to `:latest` 

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

**Comment by cardil** [comment]
/lgtm

Let's try. Github registry shouldn't be so problematic with quotas as Docker Hub.

File: test/config/monitoring.yaml [hunk 1]
```diff
@@ -43,7 +43,7 @@ spec:
     spec:
       containers:
       - name: zipkin
-        image: docker.io/openzipkin/zipkin:2.13.0
+        image: ghcr.io/openzipkin/zipkin:latest
```
Comment (cardil) [inline test/config/monitoring.yaml:46]: Maybe safer would be to use: `ghcr.io/openzipkin/zipkin:2` instead? If they release zipkin 3 that breaks something, this should save us from failure.

Comment (matzew) [reply test/config/monitoring.yaml:46]: ah, ok - that's good

will update

**Review by cardil — APPROVED** [review]
/lgtm

**Pull request merged**